app/pages/task-list/taskListManifest.json

```diff
@@ -81,6 +81,38 @@
           ]
         }
       ]
+    },
+    {
+      "name": "Add Associated Services",
+      "sections": [
+        {
+          "id": "associated-services",
+          "title": "Add Associated Services to your order",
+          "dependencies": [
+            {
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients"],
+              "zeroCountDependencies": ["service-recipients"]
+            },
+            {
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients","catalogue-solutions"],
+              "zeroCountDependencies": ["catalogue-solutions"],
+              "countDependencies": ["service-recipients"]
+            },
+            { 
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients","catalogue-solutions","additional-services"],
+              "zeroCountDependencies": ["additional-services"],
+              "countDependencies": ["service-recipients","catalogue-solutions"]
+            },
+            { 
+              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients","catalogue-solutions","additional-services"],
+              "countDependencies": ["service-recipients","catalogue-solutions","additional-services"]
+            },
```

Comment:
ok little confused I thought there are 2 combos it is enabled. One when service recipient is 0
```
{
              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients"],
              "zeroCountDependencies": ["service-recipients"]
            },
```
Which is what you have.

And then the other combo is when service recipient is > 0 then we need catalogue-solutions and additional services as complete for it to be enabled

```
{ 
              "statusDependencies": ["description", "ordering-party", "supplier", "commencement-date", "service-recipients","catalogue-solutions","additional-services"],
              "countDependencies": ["service-recipients",]
            },
```
^ this was the only other combo I was expecting for the section to be enabled

Reply:
That is for AC 7 

Scenario 7: Additional Service now complete, >= 1 Service Recipient, >=1 Catalogue Solution, =>1 Additional Service

Its says the counts must be grater than zero ?

Reply:
Changed as advised above